common/client/node_lifecycle.go

```diff
@@ -483,14 +484,6 @@ func (n *node[CHAIN_ID, HEAD, RPC]) syncingLoop() {
 			lggr.Infow(fmt.Sprintf("Successfully verified RPC node. Node was syncing for %s", time.Since(syncingAt)), "nodeState", n.State())
 			n.declareAlive()
 			return
-		case <-time.After(zombieNodeCheckInterval(n.noNewHeadsThreshold)):
```

Comment:
What was the reason for removing this?
Are we now not forcing to use a syncing node if that's the only node?

Reply:
Following [your suggestion](https://github.com/smartcontractkit/chainlink/pull/11765#discussion_r1460542437), I've run some tests, and it turns out that RPC is not able to properly serve requests while it's syncing and marking such node as Alive leads to a lot of error noise.